src/_rules/color.js

```diff
@@ -1,5 +1,15 @@
-import { handler as h } from '#utils';
+
+const handleOpacity = ([, d = ''], { theme }) => {
```

Comment:
Delete this in favor of the change proposed below?

Reply:
Yes, fixing